package.json

```diff
@@ -3,13 +3,15 @@
 	"scripts": {
 		"build": "lerna run build && lerna run build:types",
 		"compile": "lerna run compile",
-		"checkFormat": "npm run prettier -- --list-different",
+		"checkFormat": "npm run prettier -- --list-different && npm run lint",
 		"coveralls": "node node_modules/coveralls/bin/coveralls.js < coverage/lcov.info",
-		"format": "npm run prettier -- --write",
+		"format": "npm run prettier:write && npm run lint:fix",
 		"lerna": "lerna bootstrap -- --no-optional --no-package-lock",
 		"link": "lerna run link",
 		"lint": "eslint '**/*.{js,ts,tsx}'",
-		"prettier": "prettier-eslint '**/*.{js,ts,tsx}'",
+		"lint:fix": "eslint '**/*.{js,ts,tsx}' --fix",
+		"prettier": "prettier '**/*.{js,ts,tsx}'",
+		"prettier:write": "prettier '**/*.{js,ts,tsx}' --write",
```

Comment:
For consistency, you might want to change this to match some other projects, we have `prettier` (writes) and `prettier:check` (just reads/checks): rationale — as Prettier is just an pretty-printer (no AST changes) it is totally safe to run, so we make the shorter command (the default), do the most useful and safe thing. This is in contrast to ESLint, where the making "fix" the default wouldn't be safe because the changes it makes can be quite aggressive, so we make the shorter command (`eslint`) just check but not write.

Projects following this pattern are [alloy-editor](https://github.com/liferay/alloy-editor/blob/5795cf638525f1d37cb84f92dffcab7e5452b92c/package.json#L60-L66), [eslint-config-liferay](https://github.com/liferay/eslint-config-liferay/blob/b056a199ef1f239e43cd74c3980d8ac47ba5231e/package.json#L48-L51), [liferay-npm-tools](https://github.com/liferay/liferay-npm-tools/blob/40f5d3cd148ce007231a333bcb8f55cd28c47243/package.json#L15-L18), [liferay-js-themes-toolkit](https://github.com/liferay/liferay-js-themes-toolkit/blob/c95eca5fb6330f507f87266bd8486927cc893897/package.json#L15-L18), [liferay-js-toolkit](https://github.com/liferay/liferay-js-toolkit/blob/3004c05a99ff0252fd29759ae25efd922fd6e154/package.json#L14-L18), [liferay-amd-loader](https://github.com/liferay/liferay-amd-loader/blob/8b1f2a2ff563a6b279649e7121906f3ba4a69f21/package.json#L12-L15), which is to say, most of the ones that are under active development.